### PR TITLE
add interpolated until field

### PR DESCRIPTION
**Pull request by lodamar**

### CODE REVIEW

File: src/main/scala/it/bitrock/dvs/api/model/model.scala [hunk 1]
```diff
@@ -32,7 +32,8 @@ final case class FlightReceived(
     airline: Airline,
     airplane: Airplane,
     status: String,
-    updated: Long
+    updated: Long,
+    interpolatedUntil: Long
```
Comment (massimosiani) [inline src/main/scala/it/bitrock/dvs/api/model/model.scala]: the frontend should not notice the difference, so I believe that we should replace updated with interpolatedUntil and remove the last one

Comment (lodamar) [reply src/main/scala/it/bitrock/dvs/api/model/model.scala]: Working on this, as soon as tests pass I'll update the PR

File: src/test/scala/it/bitrock/dvs/api/kafka/FlightListKafkaConsumerSpec.scala [hunk 2]
```diff
@@ -13,8 +13,8 @@ import it.bitrock.dvs.model.avro.{
   AirlineInfo => KAirlineInfo,
   AirplaneInfo => KAirplaneInfo,
   AirportInfo => KAirportInfo,
-  FlightReceived => KFlightReceived,
-  FlightReceivedList => KFlightReceivedList,
+  FlightInterpolated => KFlightReceived,
```
Comment (massimosiani) [inline src/test/scala/it/bitrock/dvs/api/kafka/FlightListKafkaConsumerSpec.scala]: do we need to rename those two?

File: src/main/scala/it/bitrock/dvs/api/model/AvroConverters.scala [hunk 3]
```diff
@@ -6,8 +6,8 @@ import it.bitrock.dvs.model.avro.{
   AirportInfo => KAirportInfo,
   CountAirline => KCountAirline,
   CountFlight => KCountFlight,
-  FlightReceived => KFlightReceivedEvent,
-  FlightReceivedList => KFlightReceivedListEvent,
+  FlightInterpolated => KFlightReceivedEvent,
```
Comment (massimosiani) [inline src/main/scala/it/bitrock/dvs/api/model/AvroConverters.scala]: do we need to rename them?

File: src/test/scala/it/bitrock/dvs/api/kafka/FlightListKafkaConsumerSpec.scala (hunk 2 above)
Comment (lodamar) [reply src/test/scala/it/bitrock/dvs/api/kafka/FlightListKafkaConsumerSpec.scala]: good point, gonna remove them

File: src/main/scala/it/bitrock/dvs/api/model/AvroConverters.scala (hunk 3 above)
Comment (lodamar) [reply src/main/scala/it/bitrock/dvs/api/model/AvroConverters.scala]: same as above

**Pull request merged**